packages/abstract-document/src/abstract-document-exporters/pdf/render.ts

```diff
@@ -413,16 +423,21 @@ function drawText(
   text: string,
   alignment: AD.TextStyle.TextAlignment
 ) {
+  //compensation needed as pdfKit's widthOfString may return a slightly
+  //lower value than the actual size due to loss of precision
+  const compensatePdfKitSize = 1;
   const font = getFontName(textStyle);
   const fontSize = AD.TextStyle.calculateFontSize(textStyle, 10);
   const offset = calculateTextOffset(textStyle, fontSize);
+  const alignmentOffset = calculateTextAlignmentOffset(alignment, compensatePdfKitSize);
+
   pdf
     .font(font)
     .fontSize(fontSize)
     .fillColor(textStyle.color || "black")
-    .text(text, finalRect.x, finalRect.y + offset, {
-      width: finalRect.width + 1,
-      height: finalRect.height,
+    .text(text, finalRect.x - alignmentOffset, finalRect.y + offset, {
+      width: finalRect.width + compensatePdfKitSize,
```

Comment:
Shouldn't this be compensatePdfKitSize * 0.5 for "center" alignment? Otherwise the text will be moved slightly to the right when center aligned.

Reply:
The changes to the width should already be compensated for in the alignmentOffset when using alignment "center". For example, if the original width is 10, we add the compensation and it becomes 11, if the alignment is "center" we then remove half of the compensation from the x position. This will cause the center of the text to still be centered as before, while still having more width so we dont prematurely linebreak it. (i also tested it with a compensation of 10 and it looked good)